circle.yml

```diff
@@ -9,19 +9,27 @@ workflows:
 jobs:
   test:
     docker:
-      - image: circleci/node:8
+      - image: cypress/base:8
 
     steps:
       - checkout
-
+      
+      - restore_cache:
```

Comment:
I suggest we start using https://circleci.com/orbs/registry/orb/circleci/node because it simplifies installing and caching a lot

Reply:
or even our Cypress orb